cluster-api/src/main/java/io/aiven/klaw/clusterapi/services/ApacheKafkaTopicService.java

```diff
@@ -0,0 +1,220 @@
+import io.aiven.klaw.clusterapi.utils.ClusterApiUtils;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.CreateTopicsResult;
+import org.apache.kafka.clients.admin.DeleteTopicsResult;
+import org.apache.kafka.clients.admin.DescribeTopicsResult;
+import org.apache.kafka.clients.admin.ListTopicsOptions;
+import org.apache.kafka.clients.admin.ListTopicsResult;
+import org.apache.kafka.clients.admin.NewPartitions;
+import org.apache.kafka.clients.admin.NewTopic;
+import org.apache.kafka.clients.admin.TopicDescription;
+import org.apache.kafka.common.KafkaException;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+public class ApacheKafkaTopicService {
+
+  private static final long TIME_OUT_SECS_FOR_TOPICS = 5;
+
+  private Environment env;
+
+  private ClusterApiUtils clusterApiUtils;
+
+  public ApacheKafkaTopicService(Environment env, ClusterApiUtils clusterApiUtils) {
+    this.env = env;
+    this.clusterApiUtils = clusterApiUtils;
+  }
+
+  public synchronized Set<HashMap<String, String>> loadTopics(
+      String environment, KafkaSupportedProtocol protocol, String clusterName) throws Exception {
+    log.info("loadTopics {} {}", environment, protocol);
+    AdminClient client = clusterApiUtils.getAdminClient(environment, protocol, clusterName);
+    Set<HashMap<String, String>> topics = new HashSet<>();
+    if (client == null) {
+      throw new Exception("Cannot connect to cluster.");
+    }
+
+    ListTopicsOptions listTopicsOptions = new ListTopicsOptions();
+    listTopicsOptions = listTopicsOptions.listInternal(false);
+
+    ListTopicsResult topicsResult = client.listTopics(listTopicsOptions);
+
+    try {
+      DescribeTopicsResult s = client.describeTopics(new ArrayList<>(topicsResult.names().get()));
+      Map<String, TopicDescription> topicDesc =
+          s.all().get(TIME_OUT_SECS_FOR_TOPICS, TimeUnit.SECONDS);
+      Set<String> keySet = topicDesc.keySet();
+      keySet.remove("_schemas");
+      List<String> lstK = new ArrayList<>(keySet);
+      HashMap<String, String> hashMap;
+      for (String topicName : lstK) {
+        hashMap = new HashMap<>();
+        hashMap.put("topicName", topicName);
+        hashMap.put(
+            "replicationFactor",
+            "" + topicDesc.get(topicName).partitions().get(0).replicas().size());
+        hashMap.put("partitions", "" + topicDesc.get(topicName).partitions().size());
+        if (!topicName.startsWith("_confluent")) {
+          topics.add(hashMap);
+        }
```

Comment:
```suggestion
      Map<String, String> hashMap;
      for (String topicName : lstK) {
        if (topicName.startsWith("_confluent")) continue;
        hashMap = new HashMap<>();
        hashMap.put("topicName", topicName);
        hashMap.put(
            "replicationFactor",
            "" + topicDesc.get(topicName).partitions().get(0).replicas().size());
        hashMap.put("partitions", "" + topicDesc.get(topicName).partitions().size());
        topics.add(hashMap);
        }
```

This one i didn't get... it seems that each iteration we create a map, however if a topic name starts with `_confluent`  we just do nothing with this new map... So to avoid these useless maps creation i would suggest just skip current iteration if a topic names starts with `_confluent`